multiversx_sdk_cli/projects/report/report_creator.py

```diff
@@ -47,7 +52,7 @@ def _create_project_report(self, parent_path: Path, project_path: Path) -> Proje
         project = load_project(project_path)
 
         if not self.skip_build:
-            project.build(self.build_options)
+            project.build(self.build_args, self.build_options)
```

Comment:
I think we can remove `get_option()` from `ProjectBase`, `ProjectRust` etc. If so, can we also remove the "options" property and possibly the build options args? (I did not completely check this).

Reply:
I don't think we can. I deleted `get_options()` as it wasn't used but when building the contract we have the following line:
`with_wasm_opt = not self.options.get("no-wasm-opt")`. I think both `options` and `build_args` are needed.